net-irc/weechat/metadata.xml

```diff
@@ -9,6 +9,10 @@
 		<email>zoltan@sinustrom.info</email>
 		<name>Zoltan Puskas</name>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
```

Comment:
Hmm, I thought we had a rule that the proxy-maint project should also be added if a proxied maintainer was added.

Reply:
Me too, but... https://github.com/gentoo/gentoo/pull/33064#discussion_r1337426042

Reply:
Yes, there it is: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Could you add the proxy-maint project too please?

Reply:
Done